module/migration.mjs

```diff
@@ -522,6 +523,29 @@ function _migrateActorAC(actorData, updateData) {
 
 /* -------------------------------------------- */
 
+/**
+ * Migrate the actor movement & senses to replace `0` with `null`.
+ * @param {object} actorData   Actor data being migrated.
+ * @param {object} updateData  Existing updates being applied to actor. *Will be mutated.*
+ * @returns {object}           Modified version of update data.
+ * @private
+ */
+function _migrateActorMovementSenses(actorData, updateData) {
+  if ( foundry.utils.isNewerVersion("2.4.0", actorData._stats.systemVersion) ) {
```

Comment:
Let's just guard this in case we're sent a partial actor delta or something.

```suggestion
  if ( actorData?._stats?.systemVersion && foundry.utils.isNewerVersion("2.4.0", actorData._stats.systemVersion) ) {
```